File: src/main/Java/boj_step4/boj10871.java
import java.io.*;
import java.util.StringTokenizer;

public class boj10871 {
    public static void main(String[] args) throws IOException {
        BufferedReader reader = new BufferedReader(new InputStreamReader(System.in));
        BufferedWriter writer = new BufferedWriter(new OutputStreamWriter(System.out));
        StringTokenizer tokenizer = new StringTokenizer(reader.readLine());
        int n = Integer.parseInt(tokenizer.nextToken());
        int x = Integer.parseInt(tokenizer.nextToken());
        tokenizer = new StringTokenizer(reader.readLine());
        int[] list = new int[n];
        for (int i = 0; i < n; i++) list[i] = Integer.parseInt(tokenizer.nextToken());
        for (int it: list) { if (it < x) writer.append(it + " "); }
        writer.close();
    }
}

File: src/main/kotlin/boj10871.kt
import java.io.BufferedReader
import java.io.BufferedWriter
import java.io.InputStreamReader
import java.io.OutputStreamWriter
import javax.swing.ViewportLayout

fun main() {
    val r = BufferedReader(InputStreamReader(System.`in`))
    val w = BufferedWriter(OutputStreamWriter(System.out))
    val (n, x) = r.readLine()!!.split(" ").map { it.toInt() }
    val arr = r.readLine()!!.split(" ").map { it.toInt() }
    for(i in 0 until n) if(arr[i] < x) w.write("${arr[i]} ")
    w.close()
}